com.unity.netcode.gameobjects/Tests/Editor/StartStopTests.cs

```diff
@@ -0,0 +1,62 @@
+using NUnit.Framework;
+using UnityEngine;
+
+namespace Unity.Netcode.EditorTests
+{
+    public class StartStopTests
+    {
+        private NetworkManager m_NetworkManager;
+
+        [SetUp]
+        public void Setup()
+        {
+            // Create the reusable NetworkManager
+            m_NetworkManager = new GameObject(nameof(NetworkManager)).AddComponent<NetworkManager>();
+            var transport = m_NetworkManager.gameObject.AddComponent<DummyTransport>();
+
+            m_NetworkManager.NetworkConfig = new NetworkConfig()
+            {
+                NetworkTransport = transport
+            };
+        }
+
+        [Test]
+        public void TestStopAndRestartForExceptions()
```

Comment:
These look much better, good job!

nit: The team just had a conversation about this, and still isn't aligned quite yet so feel free to disregard, but I'm a fan of more descriptive test names... e.g. either SUTShouldDoExpectedThingWhenSpecificState or SUT_SpecificState_ExpectedResult so in this case it might be something like NetworkManagerShouldNotThrowExceptionsWhenRestarted, or NetworkManager_StartAfterShutdown_DoesNotThrow